src/js/components/Diagram/stories/Progressing.js

```diff
@@ -72,13 +72,17 @@ export const Progressing = () => {
               ))}
             </Box>
           </Box>
-          <Diagram connections={connections} />
+          <Diagram animation connections={connections} />
```

Comment:
Is the intent for the animation to re-draw everything each time a new connection is added? does the functionality support only "drawing" the new connections?

Reply:
I think maybe as an enhancement down the line we could have a prop to serve as an option to either re-draw the lines on each new connection or keep all the connections made until a reload of the Diagram. Can definitely see use cases for both.